### ISSUE TITLE
[BUG] Custom commands output doesn't show slashes

### ISSUE THREAD

**Issue by Carghaez** [ForkIssues/TrackerWin#1207]
Hello guys! 

I noticed that if you try to echo the path it is impossible to display the slash or backslash characters.

is it something wanted so?

![image](https://user-images.githubusercontent.com/297838/122818549-a4d39d00-d2d9-11eb-9fe9-8166cb5d8767.png)

**Comment by DanPristupov** [comment]
Can you write steps to reproduce the problem?

**Comment by Carghaez** [comment]
Yes, sure, on windows machine create a custom command as described below

![image](https://user-images.githubusercontent.com/297838/122824807-6cd05800-d2e1-11eb-9046-904b75d9b4f9.png)

![image](https://user-images.githubusercontent.com/297838/122824925-8c678080-d2e1-11eb-9afa-96536e5df653.png)

the results is the path without slash or backslash

![image](https://user-images.githubusercontent.com/297838/122825042-b751d480-d2e1-11eb-8660-f709910ddf28.png)

The path is S:\Dev\workspaces\....
